telcell/celldb/__main__.py

```diff
@@ -0,0 +1,84 @@
+import sys
+from functools import partial
+from typing import Sequence
+
+import click
+from tqdm import tqdm
+
+from telcell.celldb import pgdatabase
+from telcell.utils import script_helper
+
+
+def create_antenna_light_table(con):
+    pgdatabase.create_table(con)
+    tablename = "antenna_light"
+    with con.cursor() as cur:
+        cur.execute(
+            f"""
+            INSERT INTO {tablename}(date_start, date_end, radio, mcc, mnc, lac, ci, eci, rd, azimuth)
+            SELECT start, "end",
+                CASE
+                    WHEN radio = '2G' THEN 'GSM'
+                    WHEN radio = '3G' THEN 'UMTS'
+                    WHEN radio = '4G' THEN 'LTE'
+                    WHEN radio = '5G' THEN 'NR'
+                    ELSE radio
+                END radio,
+                204, mnc,
+                lac,
+                CASE
+                    WHEN radio = '4G' THEN NULL
+                    ELSE cell
+                END ci,
+                CASE
+                    WHEN eci is NOT NULL THEN eci
+                    WHEN radio = '4G' AND cell is NOT NULL THEN cell
+                    ELSE NULL
+                END eci,
+                'SRID=4326;POINT('||rdx||' '||rdy||')', azimuth
+            FROM antenna
```

Comment:
nog even ergens (in de readme ofzo?) dat als je dus zelf een database hebt, er verwacht wordt dat je een antenne tabel hebt, en met welke kolommen. Of als je nog een csv moet inlezen, waar die csv dan allemaal aan moet voldoen